sonaranalyzer-dotnet/rspec/cs/S3897_c#.html

```diff
@@ -5,27 +5,13 @@
 <h2>Noncompliant Code Example</h2>
 <pre>
 class MyClass  // Noncompliant
-{
-  public override bool Equals(object other)
```

Comment:
MyClass does not implement IEquatable. Why did you remove this example?

Reply:
It follows the discussion here: https://github.com/SonarSource/sonar-dotnet/issues/420
Implementation was already updated not to raise issue in that case, only the code examples were not up-to-date